gnucash/gtkbuilder/dialog-customer.glade

```diff
@@ -240,7 +240,7 @@
                             <property name="visible">True</property>
                             <property name="can_focus">False</property>
                             <property name="halign">end</property>
-                            <property name="label" translatable="yes">Name</property>
+                            <property name="label" translatable="yes">Person</property>
```

Comment:
No bots allowed any longer? ;-)

Reply:
Or departments or mail stops...

Like I said, it's correct as-is for US English. Translate it how you want but leave the English alone; add context to distinguish it from other possibly different uses of the word. Yes, I understand that increases the load on some translators but without the context it might be impossible to correctly translate into some languages.